package.json

```diff
@@ -46,6 +46,7 @@
 		"onLanguage:shellscript",
 		"onLanguage:typescript",
 		"onLanguage:sparql",
+		"onLanguage:scm",
```

Comment:
`languageId` is `scm` but the nvim-treesitter package is called `query`.

![image](https://user-images.githubusercontent.com/1354217/171777594-e9ebe264-7d75-4f74-a891-36a7788761e1.png)